lib/modules/dosomething/dosomething_campaign/includes/Campaign.php

```diff
@@ -444,6 +455,49 @@ protected function getMobileCommonsData() {
     ];
   }
 
+  protected function getPartners() {
+    $data = [];
+
+    // @TODO: only accounts for single partner at the moment.
+    // Update extract function to grab array of partners instead of just first id.
```

Comment:
Yeah, sure. But I find issues are sometimes as susceptible to being forgotten. Would also like to leave the `@TODO` cause then it reminds us when we're inside the code. I could create an issue and reference it in the todo comment as well?